backend/compact-connect/lambdas/python/common/tests/function/test_data_client.py

```diff
@@ -830,12 +831,14 @@ def test_deactivate_privilege_on_inactive_privilege_has_no_effect(self):
         test_data_client = DataClient(self.config)
 
         # Now, deactivate the privilege
-        test_data_client.deactivate_privilege(
-            compact='aslp',
-            provider_id=provider_id,
-            jurisdiction='ne',
-            license_type_abbr='aud',
-        )
+        with self.assertRaises(CCInvalidRequestException) as context:
```

Comment:
I guess this will make more sense as an interface when we start accepting descriptions on a deactivation - admins will expect those to show up, so idempotency is less useful. 👍